tests/test_sync.py

```diff
@@ -311,3 +312,27 @@ def test_testing_mode_context():
         assert 3 == attempt.num
 
     assert not stamina.is_testing()
+
+
+def test_compute_backoff_uses_logarithm():
+    """Test _compute_backoff that short circuits to using log."""
```

Comment:
can you please rephrase the docstrings according to
https://jml.io/test-docstrings/ and leave the `"""` an own line each?

In this case it would be something like

> If $MY_MAGIC_FORMULA detects an opportunity to short-circuit, max_backoff is returned.

Except in a way that it makes sense – c.f. also my request for an explanation of the formula.